src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
Oh I see now. I'm not sure. I would prefer to move it to common when it's actually used in common.

Today there are a couple similar mechanisms built-in to the infrastructure already. Within the worker project, when a variable is accessed (executionContext.Variables.Get/Set), the value is traced to the diag logs. Similarly within the task libs, when a variable is accessed, it is traced as debug output to the job log.